Screens/Profile.tsx

```diff
@@ -0,0 +1,283 @@
+                handleSubmit,
+                setFieldValue,
+                resetForm,
+                values,
+                errors,
+            }) => (
+                <View style={styles.container}>
+                    <StatusBar />
+                    <ImageBackground
+                        style={styles.hero}
+                        source={{
+                            uri: "https://media.istockphoto.com/id/1420755455/photo/grunge-dirty-background-overlay.jpg?b=1&s=170667a&w=0&k=20&c=7wfOkV1I0RJVAsuZlCzMD8zRenjlWo1hUYCAIsGBckw=",
+                        }}
+                    >
+                        <Image
+                            source={require("./Images/profile-placeholder.png")}
+                            style={styles.profileImg}
+                        />
+
+                        <View style={styles.textLink}>
+                            <CustomButton
+                                title="edit profile"
+                                buttonWidth={100}
+                                buttonOnPress={() =>
+                                    setModalVisible(!modalVisible)
+                                }
+                            />
+                        </View>
+                    </ImageBackground>
+
+                    <ProfileField
+                        label={"Name "}
+                        value={initalFormValues.name}
+                        icon="name"
+                    />
+
+                    <Seperator />
+
+                    <ProfileField
+                        label={"Email"}
+                        value={initalFormValues.email}
+                        icon="email"
+                    />
+                    <Seperator />
+
+                    <ProfileField
+                        label={"Phone"}
+                        value={initalFormValues.phone}
+                        icon="phone"
+                    />
+
+                    <Seperator />
+
+                    <Modal
+                        animationType="slide"
+                        transparent={true}
+                        visible={modalVisible}
+                        onRequestClose={() => {
+                            setModalVisible(!modalVisible);
+                        }}
+                    >
+                        <View style={styles.centeredView}>
+                            <View style={styles.modalView}>
+                                <Text style={styles.title}> Edit Profile</Text>
+                                <View>
+                                    <ImageBackground
+                                        resizeMode={"cover"}
+                                        style={styles.hero}
+                                        source={{
+                                            uri: "https://media.istockphoto.com/id/1420755455/photo/grunge-dirty-background-overlay.jpg?b=1&s=170667a&w=0&k=20&c=7wfOkV1I0RJVAsuZlCzMD8zRenjlWo1hUYCAIsGBckw=",
```

Comment:
you should probaby add this image to the Images folder to match the one below instead of having one a uri and one in the folder

Reply:
👍🏻